mangle_data.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+import fileinput
+from urllib import request
+
+parser = argparse.ArgumentParser(
+    description="Adjust a line from wb-cli so it can be fed into the target wiki"
+)
+
+parser.add_argument("-t", "--target", action="store", dest="target", required=True)
+parser.add_argument("-p", "--pick", action="append", dest="pick", required=True)
+
+
+def process_line(line, pick=[], select_languages=set()):
+    data = json.loads(line)
+    out = {}
+    for key, value in data.items():
+        if key not in pick:
+            continue
+
+        if not isinstance(value, dict):
+            out[key] = value
```

Comment:
so this is basically "skip if there are no <labels|descriptions>" etc?

Reply:
Yes.